app/controllers/Api.scala

```diff
@@ -69,7 +80,16 @@ class Api @Inject() (val dataStore: DataStore,
 
         dataStore.updateMediaAtom(newAtom).fold(
           err => InternalServerError(err.msg),
-          _ => Created(s"updated atom $atomId").withHeaders("Location" -> atomUrl(atom.id))
+          _ => {
+            
+            val event = ContentAtomEvent(atom, EventType.Update, now())
```

Comment:
I think this should be `newAtom`? Current it is publishing the state of the atom before the asset is added.